mllib/pom.xml

```diff
@@ -63,25 +63,20 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.scalanlp</groupId>
```

Comment:
Please keep the breeze dependency in MLlib's pom because we call its APIs directly. We can move versions to a common place.